test/simple/test-fs-write-file.js

```diff
@@ -97,12 +97,46 @@ fs.writeFile(filename3, n, { mode: m }, function(e) {
   });
 });
 
+// test that writeFile accepts file descriptors
+var filename4 = join(common.tmpDir, 'test4.txt');
+var buf = new Buffer(s, 'utf8');
+common.error('writing to ' + filename4);
+
+fs.open(filename4, 'w+', function(e, fd) {
+  if (e) throw e;
+
+  ncallbacks++;
+  common.error('file4 opened');
+
+  fs.writeFile(fd, buf, function(e) {
```

Comment:
It seems that we're missing a test for `fs.writeFileSync(fd, ...)`, could you please add one? Thanks!